File: JAVApb5/ConcursModel/src/main/java/concurs/domain/ProbaDTO.java
package concurs.domain;

import java.io.Serializable;

public class ProbaDTO extends  Proba implements Serializable {
    private int nrParticipanti;


    public ProbaDTO(String denumire, int varstaMin, int varstaMax,int nrParticipanti) {
        super(denumire, varstaMin, varstaMax);
        this.nrParticipanti=nrParticipanti;
    }

    @Override
    public String toString() {
        return "ProbaDTO{" +super.toString()+
                "nrParticipanti=" + nrParticipanti +
                '}';
    }

    public int getNrParticipanti() {
        return nrParticipanti;
    }

    public void setNrParticipanti(int nrParticipanti) {
        this.nrParticipanti = nrParticipanti;
    }
}

File: CSharp/CSharp/model/ProbaDTO.cs
using System;

namespace model
{     [Serializable]
    public class ProbaDTO:Proba
    {
        public int nrParticipanti{ get; set; }

        public ProbaDTO(string denumire, int varstaMin, int varstaMax, int NrParticipanti) : base(denumire, varstaMin, varstaMax)
        {
            this.nrParticipanti = NrParticipanti;
        }

       /* public ProbaDTO(int NrParticipanti)
        {
            this.nrParticipanti = NrParticipanti;
        }*/

        public override bool Equals(object obj)
        {
            return base.Equals(obj);
        }

        public override int GetHashCode()
        {
            return base.GetHashCode();
        }

        public override string ToString()
        {
           // return base.ToString();
            return string.Format("[Proba: Id={0}, Denumire={1},VarstaMin={2},VarstaMax={3},nrParticipanti={4}]", Id, Denumire,VarstaMin,VarstaMax,nrParticipanti);
        }
    }
}